geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/hash/AbstractHScanIntegrationTest.java

```diff
@@ -453,7 +453,7 @@ public void should_notErrorGivenNonzeroCursorOnFirstCall() {
     ScanResult<Map.Entry<String, String>> result = jedis.hscan("colors", "5");
 
     assertThat(new HashSet<>(result.getResult()))
-        .containsExactlyInAnyOrderElementsOf(data.entrySet());
+        .isSubsetOf(data.entrySet());
```

Comment:
The behavior of hscan when you start with a non-zero cursor is undefined, except that it should not return an error. So we can only reasonably expect that we might see some but not necessarily all of the elements of the hash.